odoo/tests/common.py

```diff
@@ -311,6 +311,14 @@ class BaseCase(TreeCase, MetaCase('DummyCase', (object,), {})):
     longMessage = True      # more verbose error message by default: https://www.odoo.com/r/Vmh
     warm = True             # False during warm-up phase (see :func:`warmup`)
 
+    def setUp(self):
+        from odoo.addons.iap.tools import iap_tools
```

Comment:
I'm not sure if I should put the import here or at the top. I also thought I would need to add a `try ... catch ImportError` to handle the case where the IAP module isn't installed, but it didn't seem to be an issue when I tested it when it was uninstalled.